rpcs3/util/shared_ptr.hpp

```diff
@@ -574,7 +574,6 @@ namespace stx
 		}
 
 		// Random checks which may fail on invalid pointer
-		ensure((reinterpret_cast<u64>(r.d()->destroy.load()) - 0x10000) >> 47 == 0);
```

Comment:
Why did you remove it? does it fail the assert?

Reply:
Yes, for bigger VAs

Reply:
We are relying on functions to be allocated at address lower than 0xffff'ffff'ffff in PPU. It's not ideal but it is how RPCS3 works as of this day.